README.md

```diff
@@ -1087,6 +1087,12 @@ IEEE Int. Conf. Computer Vision (ICCV), 2021. [PDF](https://openaccess.thecvf.co
 - <a name="Li21iccv"></a>Li, S., Feng, Y., Li, Y., Jiang, Y., Zou, C., Gao, Y.,  
 *[Event Stream Super-Resolution via Spatiotemporal Constraint Learning](https://openaccess.thecvf.com/content/ICCV2021/papers/Li_Event_Stream_Super-Resolution_via_Spatiotemporal_Constraint_Learning_ICCV_2021_paper.pdf)*,  
 IEEE Int. Conf. Computer Vision (ICCV), 2021. [Code](https://github.com/lisiqi19971013/EventStream-SR), [Dataset](https://github.com/lisiqi19971013/event-based-datasets).
+- <a name="Kai23icip"></a>Kai, D., Zhang, Y., Sun, X.,  
+*[Video Super-Resolution Via Event-Driven Temporal Alignment,](https://ieeexplore.ieee.org/abstract/document/10222922/)*,  
+IEEE International Conf. on Image Processing (ICIP), 2023. [Code](https://github.com/DachunKai/EBVSR), [PDF](https://ieeexplore.ieee.org/abstract/document/10222922/).
```

Comment:
International  -> Int.

Reply:
No need for [PDF] if it is the same link as in the title.
Also, it is preferable to use the DOI link: https://doi.org/10.1109/ICIP49359.2023.10222922 instead of the https://ieeexplore.ieee.org link